src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/CodeActionsOrder.Graph.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Refactoring.V2
+{
+    internal class Graph<T>
```

Comment:
Is this adapted from Roslyn?

Reply:
Yes, this is taken from [Extension Orderer](https://github.com/dotnet/roslyn/blob/614299ff83da9959fa07131c6d0ffbc58873b6ae/src/Workspaces/Core/Portable/Shared/Utilities/ExtensionOrderer.cs) and modified a little for the current case.

Reply:
Ok, we might want to add a comment stating as such.

Reply:
@DustinCampbell can we get that API opened up publicly? 😀